bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gsaoi/Gsaoi.java

```diff
@@ -850,11 +850,6 @@ public void setPosAngleConstraint(PosAngleConstraint newValue) {
     }
 
     @Override
-    public String getPosAngleConstraintDescriptorKey() {
-        return POS_ANGLE_CONSTRAINT_PROP.getName();
-    }
-
-    @Override
```

Comment:
Observing at the parallactic angle is only important for spectroscopy, as I understand it.